invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/theme.js

```diff
@@ -38,48 +37,64 @@ $tabElement.on("keydown", function (event) {
 });
 
 /* User profile dropdown */
-$('#user-profile-dropdown.ui.dropdown')
-  .dropdown({
-    showOnFocus: false,
-    selectOnKeydown: false,
-    action: (text, value, element) => {
-      // needed to trigger navigation on keyboard interaction
-      let path = element.attr('href');
-      window.location.pathname=path;
-    },
-    onShow: () => {
-      $('#user-profile-dropdown-btn').attr('aria-expanded', true)
-    },
-    onHide: () => {
-      $('#user-profile-dropdown-btn').attr('aria-expanded', false)
-    }
-  });
+$("#user-profile-dropdown.ui.dropdown").dropdown({
+  showOnFocus: false,
+  selectOnKeydown: false,
+  action: (text, value, element) => {
+    // needed to trigger navigation on keyboard interaction
+    let path = element.attr("href");
+    window.location.pathname = path;
+  },
+  onShow: () => {
+    $("#user-profile-dropdown-btn").attr("aria-expanded", true);
+  },
+  onHide: () => {
+    $("#user-profile-dropdown-btn").attr("aria-expanded", false);
+  },
+});
 
 /* Burger menu */
-const $burgerIcon = $('#rdm-burger-menu-icon');
-const $closeBurgerIcon = $('#rdm-close-burger-menu-icon');
+const $burgerIcon = $("#rdm-burger-menu-icon");
+const $closeBurgerIcon = $("#rdm-close-burger-menu-icon");
 
 const handleBurgerClick = () => {
-  $burgerIcon.attr('aria-expanded', true);
-  $('#invenio-nav').addClass('active');
+  $burgerIcon.attr("aria-expanded", true);
+  $("#invenio-nav").addClass("active");
   $closeBurgerIcon.trigger("focus");
-  $burgerIcon.css('display', 'none');
-}
+  $burgerIcon.css("display", "none");
+};
 
 const handleBurgerCloseClick = () => {
-  $burgerIcon.css('display', 'block');
-  $burgerIcon.attr('aria-expanded', false);
-  $('#invenio-nav').removeClass('active');
+  $burgerIcon.css("display", "block");
+  $burgerIcon.attr("aria-expanded", false);
+  $("#invenio-nav").removeClass("active");
   $burgerIcon.trigger("focus");
-}
+};
 
-$burgerIcon.on({ "click" : handleBurgerClick });
-$closeBurgerIcon.on({ "click" : handleBurgerCloseClick });
+$burgerIcon.on({ click: handleBurgerClick });
+$closeBurgerIcon.on({ click: handleBurgerCloseClick });
 
-const $invenioMenu = $('#invenio-menu');
+const $invenioMenu = $("#invenio-menu");
 
-$invenioMenu.on('keydown', (event) => {
-  if(event.key === "Escape"){
+$invenioMenu.on("keydown", (event) => {
+  if (event.key === "Escape") {
     handleBurgerCloseClick();
   }
-})
+});
+
+const headerSearchbar = document.getElementById("header-search-bar");
+let communityOption;
+if (headerSearchbar.dataset.communityUrl) {
```

Comment:
I am not sure I understand the `if`: we should simply take what is configured and inject it I guess...
```
const headerSearchbar = document.getElementById("header-search-bar");
const options = JSON.parse(....);
addMultipleSearchBarToHeader(
  headerSearchbar,
  i18next.t("Search records..."),
  options,
);
```